terraform/lib/dependabot/terraform/registry_client.rb

```diff
@@ -98,6 +80,43 @@ def headers_for(hostname)
         token = tokens[hostname]
         token ? { "Authorization" => "Bearer #{token}" } : {}
       end
+
+      def services
+        @services ||=
+          begin
+            response = http_get("https://#{hostname}/.well-known/terraform.json")
```

Comment:
We're building this `"https://#{hostname}` url in quite a few places, maybe we should extract a little method for it?

Reply:
@jurre I extracted a method called `#build_url`. WDYT?